system/jaeger-operator/templates/role.yaml

```diff
@@ -54,95 +55,84 @@ rules:
   - replicasets
   - statefulsets
   verbs:
-  - 'get'
-  - 'list'
-  - 'create'
-  - 'update'
-  - 'delete'
-  - 'watch'
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
 - apiGroups:
   - extensions
+  resources:
+  - ingresses
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+# Ingress for kubernetes 1.14 or higher
+- apiGroups:
   - networking.k8s.io
   resources:
   - ingresses
   verbs:
-  - 'get'
-  - 'list'
-  - 'create'
-  - 'update'
-  - 'delete'
-  - 'watch'
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
 - apiGroups:
   - batch
   resources:
   - jobs
   - cronjobs
   verbs:
-  - 'get'
-  - 'list'
-  - 'create'
-  - 'update'
-  - 'delete'
-  - 'watch'
-# - apiGroups:
-#   - route.openshift.io
-#   resources:
-#   - routes
-#   verbs:
-#   - 'get'
-#   - 'list'
-#   - 'create'
-#   - 'update'
-#   - 'delete'
-#   - 'watch'
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
 - apiGroups:
-  - autoscaling
+  - route.openshift.io
```

Comment:
I doubt that this has any effect. We don't use openshift and that is openshift-specific group.